src/app/components/jobs-slider/jobs-slider.component.scss

```diff
@@ -0,0 +1,109 @@
+@import '../../styles/colors';
+@import '../../styles/dimensions';
+@import '../../styles/typography';
+$job-card-border-radius: 8px;
+
+.home-jobs-slider-container {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 20px;
+  @media screen and (max-width: $md-device-width) {
```

Comment:
@media declaration should be preceded by an empty line

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#emptylinebetweenblocks)